client/src/components/modal.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+import ModalFooter from './modalFooter.jsx';
+import ModalBody from './modalBody.jsx';
+
+const Modal = styled.div`
+  @media screen and (min-width: 1060px)
+  {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    box-sizing: border-box;
+  }
+`;
+
+const ModalBody1 = styled.div`
+  @media screen and (min-width: 1060px){
+      height: calc(100% - 80px);
+      overflow-y: auto;
+      border-bottom: 1px solid rgba(217, 219, 224, 0.5);
+      position: relative;
+      padding: 20px
+  }
+`;
+
+const modal = props => (
+  <Modal>
+    <div
+      className="modal-wrapper"
+      style={{
```

Comment:
Would it be possible to add this inline style to the style component CSS you have starting on line 6.

Reply:
I tried moving it but ran into issues with the css loading.